src/test/resources/network/brightspots/rcv/test_data/dominion_no_precinct_data/dominion_no_precinct_data/BallotTypeContestManifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Nit: going off the directory structure pattern for the other dominion tests, it'd be more consistent to name the folders:
`/dominion_no_precinct/dominion_no_precinct_input_data`